lib/validators/calculating_smoking_gun_validator.rb

```diff
@@ -0,0 +1,82 @@
+module Validators
+  class CalculatingSmokingGunValidator < SmokingGunValidator
+    include Validators::Validator
+    def initialize(measures, records, test_id)
+      if measures.length > 1
+        # throw some kind of an error here because we only want to do this for
```

Comment:
not sure what we really want to do there.  I reality it should make sure
there is only 1 measure being tested but we would need to make sure the ui
and other code could handle that


On Tue, Jan 19, 2016 at 11:07 AM, okeefm <notifications@github.com> wrote:

> In lib/validators/calculating_smoking_gun_validator.rb
> <https://github.com/projectcypress/cypress/pull/298#discussion_r50131918>:
>
> > @@ -0,0 +1,82 @@
> > +module Validators
> > +  class CalculatingSmokingGunValidator < SmokingGunValidator
> > +    include Validators::Validator
> > +    def initialize(measures, records, test_id)
> > +      if measures.length > 1
> > +        # throw some kind of an error here because we only want to do this for
>
> Is there supposed to be an actual error thrown here, or is this just left
> as a hook for the UI people?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/projectcypress/cypress/pull/298/files#r50131918>.
>